cou/steps/ceph.py

```diff
@@ -144,18 +144,16 @@ async def osd_noout(model: Model, enable: bool) -> None:
     try:
         ceph_mon_apps = await _get_applications(model)
     except ApplicationNotFound as e:
-        logger.warning("%s", str(e))
-        logger.warning("Skip changing 'noout', because there's no ceph-mon applications.")
+        logger.debug("%s", str(e))
+        logger.debug("Skip changing 'noout', because there's no ceph-mon applications.")
```

Comment:
Can we consider changing log levels in a separate PR?  I know that we want to improve the output, but this seems like something we could do consistently across the application in a separate change. :)

Reply:
done